Equilobe.DailyReport.SL/ReportExecutionService.cs

```diff
@@ -153,6 +153,73 @@ public void SaveIndividualDraftConfirmation(UserConfirmationContext context)
             }
         }
 
+        public void AddScheduledExecutionInstance(JiraReport report)
+        {
+            using (var db = new ReportsDb())
+            {
+                var basingSettings = db.BasicSettings.Single(bs => bs.UniqueProjectKey == report.UniqueProjectKey);
+                var execInstance = new ReportExecutionInstance
+                {
+                    DateAdded = DateTime.Now,
+                    Scope = SendScope.OnSchedule,
+                };
+                basingSettings.ReportExecutionInstances.Add(execInstance);
+                db.SaveChanges();
+
+                execInstance.CopyPropertiesOnObjects(report.ExecutionInstance);
```

Comment:
Pai aveam nevoie de id-ul instantei. Si am zis ca decat sa il setez eu, mai bine il lasam sa il puna automat, si apoi il salvam in obiectul pe care il folosesc eu.